lib/folio_sync/archives_space/manual_updater.rb

```diff
@@ -34,8 +34,8 @@ def retrieve_and_sync_aspace_resources
       def determine_potential_hrid(resource)
         if @instance_key == 'cul'
           resource['id_0']
-        elsif @instance_key == 'barnard' && resource['user_defined']['string_1']
-          resource['user_defined']['string_1']
+        elsif @instance_key == 'barnard'
+          resource['user_defined']&.dig('string_1')
```

Comment:
```suggestion
          resource.dig('user_defined', 'string_1')
```
Same effect, but takes advantage of dig's multi-layer digging!